DDL/StoredProcedures/create_document_and_document_type_type.sql

```diff
@@ -0,0 +1,19 @@
+
+ CREATE PROCEDURE [createDocumentstTable]      
+AS
+    CREATE TABLE [DocumentTypes] (
+    [DocumentTypeID] INT IDENTITY PRIMARY KEY,
+    [DocumentType] VARCHAR(20)
+    );
+
+
+    CREATE TABLE [Documents] (
+    [DocumentID] INT PRIMARY IDENTITY,
+    [File] BINARY(MAX),
+    [AverageMark] INT,
+    [Year] DATE,
+    [RequestID] INT FOREIGN KEY REFERENCES(StudentRequests.RequestID),
+    [DocumentTypeID] INT FOREIGN KEY REFERENCES(DocumentTypes.DocumentTypeID)
```

Comment:
Can you match these columns with the ERD. 
PK	DocumentID	INT		
File	BINARY(MAX)		
UploadDate	DATE	
FK	DocumentTypeID	INT	
FK	RequestID	INT